### DISCUSSION TITLE
[V5] Request for transform cases

### DISCUSSION THREAD

**Discussion by ghost** [antvis/G6#6195] [Q&A]
从文档和代码中没有看到太多关于graph的 transforms属性的用法案例，有案例供参考一下吗？

> I haven't seen too many examples of the use of graph's transforms property from the documentation and code, are there any examples for reference?

**Comment by Aarebecca** [comment]
Since transform is a new concept, there is a threshold for users to understand and get started, so we try to avoid revealing it to users. If you really want to know, you can refer to the transform implemented inside G6

> 由于 transform 是一个新的概念，对于用户来说有理解和上手门槛，因此我们尽量避免透露给用户，如果你确实想了解，可以参考G6内部实现的 transform

**Reply by ghost** [reply]
看了下G6内部实现的transform，transform的入参数据DrawData还是G6接受到的外部传入的data，初步处理后的数据，我一开始给理解错了，当外部可以传入任意格式的数据，通过transform，将数据转换成G6格式数据的功能了。
